hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerScanError.java

```diff
@@ -32,9 +32,10 @@ public enum FailureType {
     MISSING_METADATA_DIR,
     MISSING_CONTAINER_FILE,
     MISSING_CHUNKS_DIR,
-    MISSING_CHUNK_FILE,
+    MISSING_DATA_FILE,
```

Comment:
qq: Is this for the cases when `*.block` file is missing?

Reply:
Yes. I changed the name so it is agnostic of container layout.